diff --git a/services/surfaceflinger/DisplayDevice.cpp b/services/surfaceflinger/DisplayDevice.cpp
index ce98b67..1e823ab 100644
--- a/services/surfaceflinger/DisplayDevice.cpp
+++ b/services/surfaceflinger/DisplayDevice.cpp
@@ -28,6 +28,7 @@
 #include <ui/PixelFormat.h>
 
 #include <gui/SurfaceTextureClient.h>
+#include <ui/FramebufferNativeWindow.h>
 
 #include <GLES/gl.h>
 #include <EGL/egl.h>
@@ -123,7 +124,9 @@ EGLSurface DisplayDevice::getEGLSurface() const {
 
 void DisplayDevice::init(EGLConfig config)
 {
-    ANativeWindow* const window = mNativeWindow.get();
+    sp<FramebufferNativeWindow> nativeWindow = new FramebufferNativeWindow();
+    ANativeWindow* const window = nativeWindow.get();
+    //  ANativeWindow* const window = mNativeWindow.get();
 
     int format;
     window->query(window, NATIVE_WINDOW_FORMAT, &format);
